modules/infra/control/vrf_iface.c

```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024 Christophe Fontaine
+// Copyright (c) 2025 Maxime Leroy, Free Mobile
+
+#include <gr_iface.h>
+#include <gr_loopback.h>
+
+static void iface_vrf_to_api(void * /* info */, const struct iface * /* iface */) { }
+
+static int iface_vrf_init(struct iface *iface, const void * /* api_info */) {
+	return iface_loopback_init(iface);
+}
+
+static int iface_vrf_fini(struct iface *iface) {
+	iface_loopback_fini(iface);
```

Comment:
The rename is incomplete? Could you either not rename anything or rename everything `s/loopback/vrf/g` ?